stdlib/2and3/aifc.pyi

```diff
@@ -0,0 +1,69 @@
+from types import TracebackType
+import sys
+
+class Error(Exception): ...
+
+_aifc_params = NamedTuple('_aifc_params', [('nchannels', int), ('sampwidth', int), ('framerate', int),
+                                           ('nframes', int), ('comptype', bytes), ('compname', bytes)])
+
+_F = Union[Text, IO[bytes]]
+_M = Tuple[int, int, bytes]
+
+class Aifc_read:
+    def __init__(self, f: _F) -> None: ...
+    if sys.version_info >= (3, 4):
+        def __enter__(self) -> Aifc_read: ...
+        def __exit__(self, exc_type: Optional[Type[BaseException]], exc_val: Optional[BaseException],
+                     exc_tb: Optional[TracebackType]) -> None: ...
+    def initfp(self, file: IO[bytes]) -> None: ...
+    def getfp(self) -> IO[bytes]: ...
+    def rewind(self) -> None: ...
+    def close(self) -> None: ...
+    def tell(self) -> int: ...
+    def getnchannels(self) -> int: ...
+    def getnframes(self) -> int: ...
+    def getsampwidth(self) -> int: ...
+    def getframerate(self) -> int: ...
+    def getcomptype(self) -> bytes: ...
+    def getcompname(self) -> bytes: ...
+    def getparams(self) -> _aifc_params: ...
+    def getmarkers(self) -> Optional[List[_M]]: ...
+    def getmark(self, id: int) -> _M: ...
+    def setpos(self, pos: int) -> None: ...
+    def readframes(self, nframes: int) -> str: ...
+
+class Aifc_write:
+    def __init__(self, f: _F) -> None: ...
+    def __del__(self) -> None: ...
+    if sys.version_info >= (3, 4):
+        def __enter__(self) -> Aifc_write: ...
+        def __exit__(self, exc_type: Optional[Type[BaseException]], exc_val: Optional[BaseException],
+                     exc_tb: Optional[TracebackType]) -> None: ...
+    def initfp(self, file: IO[bytes]) -> None: ...
+    def aiff(self) -> None: ...
+    def aifc(self) -> None: ...
+    def setnchannels(self, nchannels: int) -> None: ...
+    def getnchannels(self) -> int: ...
+    def setsampwidth(self, sampwidth: int) -> None: ...
+    def getsampwidth(self) -> int: ...
+    def setframerate(self, framerate: int) -> None: ...
+    def getframerate(self) -> int: ...
+    def setnframes(self, nframes: int) -> None: ...
+    def getnframes(self) -> int: ...
+    def setcomptype(self, comptype: bytes, compname: bytes) -> None: ...
+    def getcomptype(self) -> bytes: ...
+    def getcompname(self) -> bytes: ...
+    def setparams(self, params: Tuple[int, int, int, int, bytes, bytes]) -> None: ...
+    def getparams(self) -> _aifc_params: ...
+    def setmark(self, id: int, pos: int, name: bytes) -> None: ...
+    def getmark(self, id: int) -> _M: ...
+    def getmarkers(self) -> Optional[List[_M]]: ...
+    def tell(self) -> int: ...
+    def writeframesraw(self, data: Any) -> None: ...  # Actual type for data is Buffer Protocol
+    def writeframes(self, data: Any) -> None: ...
+    def close(self) -> None: ...
+
+def open(f: _F, mode: Text = ...) -> Union[Aifc_read, Aifc_write]: ...
```

Comment:
You can use overloads on Literal to return a more specific return type. Looks like it just supports r, rb, w, and wb.

Also, if f is a file object mode can be None.

Reply:
Added Literal overloads, not many other Literal references currently in the repo but I followed the other example I found as best I could.